### ISSUE TITLE
JHipster

### ISSUE THREAD

**Issue by mpro7** [lukesampson/scoop#2426]
Hey,
is there a possibility to add JHipster to one of the buckets?

**Comment by Ash258** [comment]
Isn't it only npm package??
Based on: 
1. <https://www.jhipster.tech/2018/08/14/jhipster-release-5.2.0.html#how-to-upgrade>
1. [Choco install file](https://chocolatey.org/packages/jhipster)
1. <http://gist.asciidoctor.org/?github-mraible/jhipster5-demo//README.adoc#_install_jhipster_5>

**Comment by mpro7** [comment]
Thanks for your reply @Ash258 . Any JHipster expert here? :) I think it is, but somebody more experienced should confirm it.

**Comment by atomfrede** [comment]
Yes it is :) If we can make the installation of jhipster under windows easier (we have choco already) I can provide some help.
